plugins/sequence/src/IndexedFastaAdapter/IndexedFastaAdapter.ts

```diff
@@ -28,19 +28,31 @@ export default class extends BaseSequenceAdapter {
     },
   })
 
+  private readHeader(headerLocation: FileLocation) {
+     if (headerLocation['uri'] == "/path/to/fa.header.yaml") {
+       return null
+     }
+
+     return fetch(headerLocation['uri'])
```

Comment:
instead of this fetch chain, might use `openLocation(headerLocation).readFile('utf8')` (this would work with local files on disk in addition to remote ones)

Reply:
I have made this change. Thanks for pointing it out - I wouldn't have realized this on my own.